Common/Base/OpenGL/BaseManager.h

```diff
@@ -67,27 +60,33 @@ namespace base
 		virtual void createGui();
 		virtual void destroyGui();
 
-	private:
-		void windowAdjustSettings(HWND hWnd, int width, int height, bool fullScreen);
+		virtual void setWindowMaximized(bool _value);
+		virtual bool getWindowMaximized();
+
+		virtual void setWindowCoord(const MyGUI::IntCoord& _value);
+		virtual MyGUI::IntCoord getWindowCoord();
 
-		void resizeRender(int _width, int _height);
+	private:
+		//void resizeRender(int _width, int _height);
 		bool createRender(int _width, int _height, bool _windowed);
 		void drawOneFrame();
 		void destroyRender();
+		void* convertPixelData(SDL_Surface *_SDLimage, MyGUI::PixelFormat& _MyGuiPixelFormat);
```

Comment:
Identifiers beginning with _ and a capital letter are reserved by the C++ standard. You should change it to e.g.
<pre>
SDL_Surface *_image, MyGUI::PixelFormat& _myGuiPixelFormat
</pre>